west.yml

```diff
@@ -45,9 +45,8 @@ manifest:
 
     - name: zephyr
       repo-path: zephyr
-      revision: e2e3dc0771188f2a01e3aaa792f81e6c6a611eb6
+      revision: 2bf335864cac67c6001d8f7c7ad7e903b20ee080
```

Comment:
I'll move this change to separate PR. As @marc-hb indicated, we are also dealing with signature change of the `intel_adsp_ipc_send_message()` function witch is used without PM too obviously. It would work better in terms of integration and bisecting.